.pre-commit-config.yaml

```diff
@@ -8,18 +8,26 @@
     -   id: check-docstring-first
     -   id: check-json
     -   id: check-yaml
-        # Exclude fsm cookiecutter files
         files: ^(?!.*template).*\.(yaml|yml|eyaml)$
     -   id: debug-statements
     -   id: name-tests-test
-        args: [--django]
+        args:
+        - --django
         files: ^tests/.*\.py$
     -   id: requirements-txt-fixer
     -   id: flake8
         exclude: ^docs/source/conf.py$
         language_version: python2.7
 -   repo: https://github.com/asottile/reorder_python_imports.git
-    sha: 3d86483455ab5bd06cc1069fdd5ac57be5463f10
+    sha: b022734351abe44d0b05a71a4fa1175287c59b49
     hooks:
     -   id: reorder-python-imports
         language_version: python2.7
+-   repo: local
+    hooks:
+    -   id: patch-enforce-autospec
```

Comment:
Does that work even though I'm using the `script` language instead of the `python` language?